src/lib.rs

```diff
@@ -4,6 +4,17 @@
 
 pub mod circom;
 pub mod circuit;
+pub mod cli;
 pub mod process;
 pub mod program;
 pub mod runtime;
+
+use std::path::{Path, PathBuf};
+
+pub use cli::Args;
+
+pub fn build_output(output_path: &Path, filename: &str, ext: &str) -> PathBuf {
```

Comment:
Perhaps this function could be included in some module? As to keep `lib.rs` to only deal with exports

Reply:
I can create utils.rs and put this function there, no probs:)

In general it's not bad to have functions in lib.rs as well

Reply:
That's ok! I would keep it here rather than creating a new one 😅